<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="2" total="5" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="UPSTestSuite" duration-ms="31217" started-at="2021-07-07T16:31:24 EDT" finished-at="2021-07-07T16:31:55 EDT">
    <groups>
      <group name="RegressionTest">
        <method signature="ReddifTestRunner.rediffMailAlertTest()[pri:0, instance:rediff.pack.ReddifTestRunner@36b4cef0]" name="rediffMailAlertTest" class="rediff.pack.ReddifTestRunner"/>
        <method signature="UPSTestRunner.mDemoTest2()[pri:1, instance:ups.pack.UPSTestRunner@3d921e20]" name="mDemoTest2" class="ups.pack.UPSTestRunner"/>
        <method signature="UPSTestRunner.signUpLinkTest()[pri:4, instance:ups.pack.UPSTestRunner@3d921e20]" name="signUpLinkTest" class="ups.pack.UPSTestRunner"/>
      </group> <!-- RegressionTest -->
    </groups>
    <test name="UPSRegressionTest" duration-ms="31217" started-at="2021-07-07T16:31:24 EDT" finished-at="2021-07-07T16:31:55 EDT">
      <class name="ups.pack.UPSTestRunner">
        <test-method status="PASS" signature="beforeMetod()[pri:0, instance:ups.pack.UPSTestRunner@3d921e20]" name="beforeMetod" is-config="true" duration-ms="2650" started-at="2021-07-07T16:31:24 EDT" finished-at="2021-07-07T16:31:27 EDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMetod -->
        <test-method status="PASS" signature="mDemoTest2()[pri:1, instance:ups.pack.UPSTestRunner@3d921e20]" name="mDemoTest2" duration-ms="15" started-at="2021-07-07T16:31:27 EDT" finished-at="2021-07-07T16:31:27 EDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- mDemoTest2 -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:ups.pack.UPSTestRunner@3d921e20]" name="afterMethod" is-config="true" duration-ms="312" started-at="2021-07-07T16:31:27 EDT" finished-at="2021-07-07T16:31:27 EDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="beforeMetod()[pri:0, instance:ups.pack.UPSTestRunner@3d921e20]" name="beforeMetod" is-config="true" duration-ms="1768" started-at="2021-07-07T16:31:27 EDT" finished-at="2021-07-07T16:31:29 EDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMetod -->
        <test-method status="PASS" signature="signUpLinkTest()[pri:4, instance:ups.pack.UPSTestRunner@3d921e20]" name="signUpLinkTest" duration-ms="13185" started-at="2021-07-07T16:31:29 EDT" finished-at="2021-07-07T16:31:42 EDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- signUpLinkTest -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:ups.pack.UPSTestRunner@3d921e20]" name="afterMethod" is-config="true" duration-ms="248" started-at="2021-07-07T16:31:42 EDT" finished-at="2021-07-07T16:31:42 EDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
      </class> <!-- ups.pack.UPSTestRunner -->
      <class name="rediff.pack.ReddifTestRunner">
        <test-method status="PASS" signature="beforeMetod()[pri:0, instance:rediff.pack.ReddifTestRunner@36b4cef0]" name="beforeMetod" is-config="true" duration-ms="1677" started-at="2021-07-07T16:31:42 EDT" finished-at="2021-07-07T16:31:44 EDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMetod -->
        <test-method status="FAIL" signature="rediffMailAlertTest()[pri:0, instance:rediff.pack.ReddifTestRunner@36b4cef0]" name="rediffMailAlertTest" duration-ms="6940" started-at="2021-07-07T16:31:44 EDT" finished-at="2021-07-07T16:31:51 EDT">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Please enter valid user name] but found [Please enter a valid user name]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Please enter valid user name] but found [Please enter a valid user name]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at rediff.pack.ReddifTestRunner.rediffMailAlertTest(ReddifTestRunner.java:33)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- rediffMailAlertTest -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:rediff.pack.ReddifTestRunner@36b4cef0]" name="afterMethod" is-config="true" duration-ms="4348" started-at="2021-07-07T16:31:51 EDT" finished-at="2021-07-07T16:31:55 EDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
      </class> <!-- rediff.pack.ReddifTestRunner -->
    </test> <!-- UPSRegressionTest -->
  </suite> <!-- UPSTestSuite -->
</testng-results>
